test/families/location_scale.jl

```diff
@@ -1,27 +1,33 @@
 
 @testset "interface LocationScale" begin
-    @testset "$(string(covtype)) $(basedist) $(realtype)" for basedist in [:gaussian],
+    @testset "$(string(covtype)) $(basedist) $(realtype)" for basedist in [:gaussian, :gaussian_nonstd],
         covtype in [:meanfield, :fullrank],
         realtype in [Float32, Float64]
 
         n_dims = 10
         n_montecarlo = 1000_000
 
-        μ = randn(realtype, n_dims)
-        L = if covtype == :fullrank
+        location = randn(realtype, n_dims)
+        scale = if covtype == :fullrank
             LowerTriangular(tril(I + ones(realtype, n_dims, n_dims) / 2))
         else
             Diagonal(ones(realtype, n_dims))
         end
-        Σ = L * L'
 
         q = if covtype == :fullrank && basedist == :gaussian
-            FullRankGaussian(μ, L)
+            FullRankGaussian(location, scale)
         elseif covtype == :meanfield && basedist == :gaussian
-            MeanFieldGaussian(μ, L)
+            MeanFieldGaussian(location, scale)
+        elseif covtype == :fullrank && basedist == :gaussian_nonstd
+            MvLocationScale(location, scale, Normal(realtype(3), realtype(3)))
+        elseif covtype == :meanfield && basedist == :gaussian_nonstd
+            MvLocationScale(location, scale, Normal(realtype(3), realtype(3)))
```

Comment:
Would it make sense to test also a case where the base distribution is multivariate, with a non-trivial covariance matrix? Probably just something like `c = randn(n_dims, n_dims); MvNormal(randn(n_dims), c*c')`? Some code paths (e.g. the call to `Fill` in `mean`) might work differently depending on the dimension of the underlying distribution.

Reply:
We probably never want the base distribution to be multivariate.